jira/client.py

```diff
@@ -2064,6 +2064,9 @@ def search_issues(self, jql_str, startAt=0, maxResults=50, validate_query=True,
         if fields is None:
             fields = []
 
+        if isinstance(fields, list):
```

Comment:
Minor observation: Might be better to use `if`/`elif`/`elif` construct here to avoid multiple branches executing for the same input.

i.e. for `fields==None` this will reset fields to an empty list in the first branch, and then immediately create an (unnecessary) copy of said list in the second branch.

Thanks for your contribution to the project!

Reply:
Fair point - perhaps I was trying to keep it *too* simple :) PR updated.